core/server/data/migrations/versions/4.0/09-add-members-email-change-events-table.js

```diff
@@ -0,0 +1,9 @@
+const {addTable} = require('../../utils');
+
+module.exports = addTable('members_email_change_events', {
+    id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+    member_id: {type: 'string', maxlength: 24, nullable: false, references: 'members.id', cascadeDelete: true},
+    previous_email: {type: 'string', maxlength: 191, nullable: false, unique: false, validations: {isEmail: true}},
+    email: {type: 'string', maxlength: 191, nullable: false, unique: false, validations: {isEmail: true}},
```

Comment:
`members_paid_subscription_events` uses `from_*` and `to_*` prefixes - should it be the same here for email fields?

Reply:
Sure, I can update that!

Reply:
Removed previous_email as it's superfluous for now